tools/make/diplomat-coverage/src/allowlist.rs

```diff
@@ -459,5 +459,12 @@ lazy_static::lazy_static! {
         "icu::locale::Locale::Err",
         "icu::plurals::PluralOperands::Err",
 
+        // unstable APIs
+        "icu::calendar::Date::add",
```

Comment:
suggestion: diplomat-coverage should just run rustdoc with the unstable feature filtered out, if that's easy to do.

Will help us not need to cover a lot of other APIs too.